dataactcore/utils/jobQueue.py

```diff
@@ -73,5 +65,40 @@ def generate_f_file(submission_id, job_id, interface_holder_class,
         job_manager.close()
 
 
+@celery_app.task(name='jobQueue.generate_f_file')
+def generate_f_file(submission_id, job_id, interface_holder_class,
+                    timestamped_name, upload_file_name, is_local):
+    """Write rows from fileF.generateFRows to an appropriate CSV. Here the
+    third parameter, interface_holder_class, is a bit of a hack. Importing
+    InterfaceHolder directly causes cyclic dependency woes, so we're passing
+    in a class"""
+    with job_context(interface_holder_class, job_id) as job_manager:
+        rows_of_dicts = fileF.generateFRows(job_manager.session,
+                                            submission_id)
+        header = [key for key in fileF.mappings]    # keep order
+        body = []
+        for row in rows_of_dicts:
+            body.append([row[key] for key in header])
+
+        write_csv(timestamped_name, upload_file_name, is_local, header,
+                  body)
+
+
+@celery_app.task(name='jobQueue.generate_e_file')
+def generate_e_file(submission_id, job_id, interface_holder_class,
+                    timestamped_name, upload_file_name, is_local):
+    """Write file E to an appropriate CSV. See generate_file_file for an
+    explanation of interface_holder_class"""
+    with job_context(interface_holder_class, job_id) as job_manager:
+        results = job_manager.session.\
+            query(AwardProcurement.awardee_or_recipient_uniqu).\
+            filter(AwardProcurement.submission_id == submission_id).\
+            distinct()
+        duns = [r.awardee_or_recipient_uniqu for r in results]
```

Comment:
I believe we may also need to pull DUNS numbers out of D2 (AwardFinancialAssistance), we should check with Kristine on this.